daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
+		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
```

Comment:
According to man, things after the modulename are arguments... I swear I've loaded multiple modules from the same command line.  Now completely lost..